go-controller/pkg/clustermanager/egressip_controller.go

```diff
@@ -151,15 +151,15 @@ func (eIPC *egressIPClusterController) patchReplaceEgressIPStatus(name string, s
 
 func (eIPC *egressIPClusterController) getAllocationTotalCount() float64 {
 	count := 0
-	eIPC.allocator.Lock()
-	defer eIPC.allocator.Unlock()
-	for _, eNode := range eIPC.allocator.cache {
+	eIPC.nodeAllocator.Lock()
+	defer eIPC.nodeAllocator.Unlock()
+	for _, eNode := range eIPC.nodeAllocator.cache {
 		count += len(eNode.allocations)
 	}
 	return float64(count)
 }
 
-type allocator struct {
+type nodeAllocator struct {
```

Comment:
also add a doc string for that this allocator does? :) I know we never had it before but now that you are cleaning it up.....